src/index.ts

```diff
@@ -40,7 +40,7 @@ export { RequestParams };
 export default class Api {
   public config: ApiConfig;
 
-  public absenceDays: AbsenceDays;
+  public absenceDays: AbsenceDaysEndpoint;
```

Comment:
Why is only `AbsenceDays` updated here to `AbsenceDaysEndpoint`? Didn't you rename all of them?

Reply:
Right, looks like I forgot to change them here and it still compiles as those are all default exports. Do you think we should remove the default exports? I would prefer that, I think.

Reply:
While that would be nice, it's also potentially a breaking change. @tomi-timetac , @whosshell what do you think?

Reply:
Should not be a breaking change as they are used with `timetacApi.fieldName`, right? At least not more breaking than the rename :thinking:

Reply:
Ah, yes, you're probably right on that. I was thinking it's possible to import just the endpoints directly, however the import resolves to `@timetac/js-client-library/dist/absenceDays`, which very much looks like "you shouldn't do it anyway".

Reply:
Not sure how we should do it, for me the Endpoint sufix makes no sense as its only used in this file anway and with default export you can name it whatever you want.